mobile/examples/phi-3/android/app/src/main/java/ai/onnxruntime/genai/demo/MainActivity.java

```diff
@@ -85,8 +108,29 @@ public void onClick(View v) {
                     @Override
                     public void run() {
                         try {
-                            genAIWrapper.run(promptQuestion_formatted);
-                        } catch (GenAIException e) {
+                            TokenizerStream stream = tokenizer.createStream();
```

Comment:
the ort-genai classes that implement AutoCloseable should be `close()`d or the resources will leak.
https://stackoverflow.com/questions/56112598/proper-way-to-close-an-autocloseable